antenna/ext/s3/connection.py

```diff
@@ -86,6 +90,8 @@ class S3Connection:
 
     """
 
+    KeyNotFound = KeyNotFound
```

Comment:
I'll fix the comment. I missed that.

This is a convenience so I don't have to import things elsewhere. It's something we do in socorro sometimes. It makes layers of abstraction a little easier to deal with. Since this is in the S3 code, it's going away soon. We don't do this in the GCS equivalent.